common/shlibs

```diff
@@ -4040,3 +4040,17 @@ libndr-samba4.so samba-libs-4.13.2_1
 libsmb-transport-samba4.so samba-libs-4.13.2_1
 libutil-cmdline-samba4.so samba-libs-4.13.2_1
 libwinbind-client-samba4.so samba-libs-4.13.2_1
+libwx_baseu-3.1.so.4 wxWidgets-common-3.1.4_1
```

Comment:
You should update the old wx entries, not add new ones.

Reply:
Thanks. I wanted to ask about this.